sycl/test-e2e/Plugin/level-zero-event-leak.cpp

```diff
@@ -1,12 +1,12 @@
 // REQUIRES: level_zero, level_zero_dev_kit
 //
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR="level_zero:*" ZE_DEBUG=4 %GPU_RUN_PLACEHOLDER %t.out wait 2>&1 %GPU_CHECK_PLACEHOLDER
-// RUN: env ONEAPI_DEVICE_SELECTOR="level_zero:*" ZE_DEBUG=4 %GPU_RUN_PLACEHOLDER %t.out nowait 2>&1 %GPU_CHECK_PLACEHOLDER
+// RUN: %{build} %level_zero_options -o %t.out
+// RUN: env ONEAPI_DEVICE_SELECTOR="level_zero:*" ZE_DEBUG=4 %{run} %t.out wait 2>&1 | FileCheck %s
```

Comment:
Since we have the `//REQUIRES: level_zero` already in the test, do we really need the `ONEAPI_DEVICE_SELECTOR="level_zero:*"` ?

  The other tests have foregone it and I think they are improved for the change.  Probably should remove it here.

Reply:
Yes, it should be removed.